spec/factories.rb

```diff
@@ -1,14 +1,19 @@
 FactoryBot.define do
   factory :family do
-    last_name "Liso"
-    capability 3
+    last_name { FFaker::NameIT.last_name }
+    first_name { FFaker::NameIT.first_name }
+    address { ["San Cesareo", "Zagarolo", "Colonna"].sample }
+    capability { [2, 3, 4, 5].sample }
+    animals { ["2 dogs", "5 cats", "1 horse", "ducks"].sample }
+    preferred_gender { ["Female", "Male"].sample }
```

Comment:
This raises the question of: will it be ok for the admin to add this information in Italian?
Answer can very easily be yes...

Reply:
I will do all the factories changes in my rake task!